charts/istio-authorizer/values.yaml

```diff
@@ -0,0 +1,81 @@
+## istio-authorizer image parameters
+##
+image:
+  ## Image repository
+  ##
+  repository: acp.artifactory.cloudentity.com/istio-authorizer
+
+  ## Image pull policy
+  ##
+  pullPolicy: IfNotPresent
+
+  ## Image tag (immutable tags are recommended, default is the chart appVersion)
+  ##
+  tag: ""
+
+## istio-authorizer image registry secret names as an array
+##
+imagePullSecrets:
+  - name: artifactory
+
+## String to partially override istio-authorizer.name
+##
+#nameOverride: ""
+
+## String to fully override istio-authorizer.fullname
+##
+# fullnameOverride: ""
+
+## Service account to use by the istio-authorizer
+##
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+## Issuer URL
+##
+issuerURL: "http://localhost:8443/default/system"
+
+## Secret with client_id and client_secret
```

Comment:
We should probably mention that those credentials should be grabbed from ACP.

Additionally, maybe it's a good idea to mention that you should probably create those secrets securely outside of values file (i.e. with SOPS or sealed-secrets).

Reply:
Are those credentials for Istio Gateway? If yes, we could write sth like this:

You can grab the values for both the `clientID` and the `clientSecret` parameters from your Istio Gateway settings. To find them, navigate to **ACP** > **APIs** > **Your Istio Gateway** > **Settings**. 

Additionally, it is recommended to create client secrets and client IDs outside of the `values.yaml` file. You can achieve it by, for example, using SOPS (a CLI tool that encrypts only the values but allows to see the keys), or sealed-secrets (encrypted secrets that can be created by anyone, but they can be decrypted only by a controller running in the target cluster).